src/plot/axis/axis-title.js

```diff
@@ -23,6 +23,32 @@ import React from 'react';
 import {ORIENTATION} from 'utils/axis-utils';
 
 const {LEFT, RIGHT, TOP, BOTTOM} = ORIENTATION;
+const transformation = (width, height) => ({
+  [LEFT]: {
+    x: 16,
```

Comment:
no what i'm saying is add a constant on line 24

const ADJUSTMENT_FOR_TEXT_SIZE = 16;

and then use that in your transformation function